### ISSUE TITLE
Comments cause trouble with with define hat blocks

### ISSUE THREAD

**Issue by TheLogFather** [scratchblocks/scratchblocks#221]
See demo forum post here:

https://scratch.mit.edu/discuss/topic/249100/

**Comment by Scimonster** [comment]
Opened a PR to fix it. You can now add comments, and block types get ignored (not supported for define, apparently).

**Comment by tjvr** [comment]
I wonder

**Comment by tjvr** [comment]
> not supported for define, apparently

Yeah, it turned out `::`-specs just got way too complicated for `define` hats, I think.

---

Interested in fixing the visuals as well? The comment is too high up:

<img width="285" alt="screen shot 2017-10-29 at 17 56 07" src="https://user-images.githubusercontent.com/639289/32146649-7a6e6f52-bcd2-11e7-9456-bfc4ff2c2eba.png">

The relevant code will be in `lib/model.js`; perhaps in `Script::draw`, but I could be mistaken there.
